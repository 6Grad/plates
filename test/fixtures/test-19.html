<div class="date"></div>
<div class="host">
  <div class="id"></div>
  <div class="name"></div>
</div>
<div class="guest">
  <div class="id"></div>
  <div class="name"></div>
</div>
