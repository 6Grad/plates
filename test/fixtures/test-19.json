{  
  "date" : "Sunday",
  "host" : { "id": "pol_int", "name": "Poland" }, 
  "guest" : { "id": "gre_int", "name": "Greece" } 
}
