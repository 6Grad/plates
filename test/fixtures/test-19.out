<div class="date">Sunday</div>
<div class="host">
  <div class="id">pol_int</div>
  <div class="name">Poland</div>
</div>
<div class="guest">
  <div class="id">gre_int</div>
  <div class="name">Greece</div>
</div>
